scenarios/aws/fakeintake/fakeintake.go

```diff
@@ -164,14 +164,18 @@ func fargateSvcLB(e aws.Environment, namer namer.Namer, taskDef *awsxEcs.Fargate
 
 	// Hashing fakeintake resource name as prefix for Host header
 	hostPrefix := utils.StrHash(namer.ResourceName(e.Ctx().Stack()))
-	host := hostPrefix + e.ECSFakeintakeLBBaseHost()
+	host := e.ECSFakeintakeLBBaseHost().ApplyT(func(baseHost string) (string, error) {
+		return hostPrefix + baseHost, nil
+	}).(pulumi.StringOutput)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Same here you should be able to use `pulumi.Sprintf`, should be
```suggestion
	host := pulumi.Sprintf("%s%s", hostPrefix, e.ECSFakeintakeLBBaseHost()
```